src/patient-registration/patient-registration.component.tsx

```diff
@@ -77,16 +100,45 @@ export const PatientRegistration: React.FC = () => {
     return () => abortController.abort();
   }, []);
 
+  useEffect(() => {
+    const abortController = new AbortController();
+    getPrimaryIdentifierType(abortController).then(primaryIdentifierType => {
+      getSecondaryIdentifierTypes(abortController).then(secondaryIdentifierTypes => {
```

Comment:
It looks like the second call doesn't depend on the first. Since that's the case, let's [run them in parallel](https://stackoverflow.com/a/35612484/1464495).

Reply:
Thanks for the StalkOverFlow pointer.

Reply:
Resolved